local-cli/runAndroid/runAndroid.js

```diff
@@ -79,6 +79,7 @@ function buildAndRun(args, reject) {
     if (args['flavor']) {
         gradleArgs.push('install' +
           args['flavor'][0].toUpperCase() + args['flavor'].slice(1)
+          + 'Debug'
```

Comment:
@jaggs6 If you want it in a separate line, please make it consistent with rest of the code (i.e. we have the `+` sign at the end of the line). Right now it looks inconsistent.

```js
gradleArgs.push('install' +
            args['flavor'][0].toUpperCase() +
            args['flavor'].slice(1) +
           'Debug'
)
```